templates/anthropic-dev/document_template.xml.jinja

```diff
@@ -0,0 +1,10 @@
+<documents>
+{% for file in context_files %}
+<document index="{{loop.index}}">
+<source>{{file.path}}</source>
+<document_content>
+{{ file.content }}
+</document_content>
+</document>
+{% endfor %}
+</documents>
```

Comment:
should probably do something smarter here to avoid invalidating cache the moment something changes in here... or put some warning/confirmation in front of it